bin/testObservability/helper/constants.js

```diff
@@ -0,0 +1,27 @@
+exports.consoleHolder = Object.assign({},console);
+exports.BATCH_SIZE = 1000;
+exports.BATCH_INTERVAL = 2000;
+exports.API_URL = 'https://collector-observability.browserstack.com';
```

Comment:
lets move api_url to config.js

Reply:
Discussed f2f, keeping testObservability config separate.